issue-006/solution.js

```diff
@@ -1,4 +1,15 @@
-// implement your solution here
-function selectSum() {
-
-}
\ No newline at end of file
+function selectSum(a) {
```

Comment:
This was not supposed to be in this diff, looks like you branched off the previous branch instead of `master`.